lib/reek/cli/options.rb

```diff
@@ -55,10 +57,11 @@ def color_support?
         $stdout.tty?
       end
 
-      # :reek:TooManyStatements: { max_statements: 6 }
+      # :reek:TooManyStatements: { max_statements: 7 }
```

Comment:
Justified as well. I believe we should overhaul our whole options handling but this is again something for a different PR.